### PR TITLE
CON-304 documentation changes (1.3.0)

### PR DESCRIPTION
**Pull request by Alxe**
Also, make minor fixes to Release Notes.

### CODE REVIEW

File: docs/release_notes.rst [hunk 1]
```diff
@@ -247,15 +259,6 @@ Refer to :ref:`Loading Connext DDS Add-On Libraries` for more information.
...
 ^^^^^^^^^^^^^^^^^^^^^
 
-1.0.0 is the first official release of *RTI Connector for JavaScript* as well as
-`RTI Connector for Python <https://community.rti.com/static/documentation/connector/1.0.0/api/python/index.html>`__.
-
-If you had access to previous experimental releases, this release makes the product
-more robust, modifies many APIs and adds new functionality. However the old 
-APIs have been preserved for backward compatibility as much as possible.
-
-*RTI Connector* 1.0.0 is built on `RTI Connext DDS 6.0.1 <https://community.rti.com/documentation/rti-connext-dds-601>`__.
```
Comment (elainerti) [inline docs/release_notes.rst:257]: I think we should at least keep the line that talk about connector 1.0.0 is built on connext 6.0.1

**Review by elainerti — CHANGES_REQUESTED** [review]

File: docs/release_notes.rst [hunk 2]
```diff
@@ -38,6 +32,24 @@ macOS® platforms. It has been tested on the following systems:
...
+Replaced Foreign Function Interface third-party library
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In this release, Connector has replaced ``node-ffi-napi`` with ``koffi`` as the
+Foreign Function Interface library. This change shouldn't affect your use of
+Connector, but makes the product more stable and reliable.
+
+This change addresses `Github Issue #198 <https://github.com/rticommunity/rticonnextdds-connector-js/issues/198>`_.
 
 Version 1.3.0
```
Comment (elainerti) [inline docs/release_notes.rst:54]: I think the version 1.3.0 section should be moved to the previous releases section??

Comment (rkorte) [reply docs/release_notes.rst:54]: Per Elaine, I will redo this PR against the release branch, release/connector/1.3.1.

**Review by rkorte — DISMISSED** [review]
Per Elaine, we need to recreate this change against release/connector/1.3.1. I will do that, and then we can decline this one.

File: docs/release_notes.rst (hunk 2 above)
Comment (rkorte) [reply docs/release_notes.rst:54]: I change my mind, I will ask Lissa and Adelle to work on this.

File: docs/release_notes.rst (hunk 1 above)
Comment (Alxe) [reply docs/release_notes.rst:257]: This has been moved to the bottom of the document. It was placed in an incorrect place as part of [this PR](https://github.com/rticommunity/rticonnextdds-connector-js/pull/200/files#diff-d778557532960c8ff205506cdecb469236ec50ca060b2da4c0b9843533a43f1aL239).

See the generated output outline.

image

**Review by Alxe — COMMENTED** [review]
I updated the PR with the applied feedback and a few other fixes.

File: docs/release_notes.rst [hunk 3]
```diff
@@ -38,19 +32,41 @@ macOS® platforms. It has been tested on the following systems:
 `the main Connector
 repository <https://github.com/rticommunity/rticonnextdds-connector>`__.
 
+Version 1.3.1
+=============
+
+What's New in 1.3.1
+-------------------
+
+*RTI Connector* 1.3.1 is built on *RTI Connext DDS* 7.3.0.2.
```
Comment (lloadholt) [inline docs/release_notes.rst:41]:
```suggestion
*RTI Connector* 1.3.1 is built on *RTI Connext* 7.3.0.2.
```

**Review by elainerti — APPROVED** [review]

**Review by lloadholt — APPROVED** [review]

**Pull request merged**